src/main/java/com/example/service/StudentService.java

```diff
@@ -67,7 +68,7 @@ public String getStudentFirstName(String studentLastName) {
 		
 		StudentEntity entity = studentDao.getStudentFirstName(studentLastName);
 		if(entity == null){
-			throw new RuntimeException();
+			throw new AccountFirstNameNotFoundException("student not found " + studentLastName);
```

Comment:
variable name is studentLastName, exception name has FirstName. change variable name?